neuromation/api/config.py

```diff
@@ -578,21 +580,19 @@ def _validate_user_config(
     #
     # Since currently CLI is the only API client that reads user config data, API
     # validates it.
-    #
-    # Later, after possible introduction of plugin subsystem the validation should
-    # be extended by plugin-provided rules.  That will be done by providing
-    # additional API for describing new supported config sections, keys and values.
-    # Right now this functionality is skipped for the sake of simplicity.
-    _check_sections(config, {"alias", "job", "storage"}, filename)
-    _check_section(
-        config, "job", {"ps-format": str, "life-span": str}, filename,
-    )
-    _check_section(
-        config,
-        "storage",
-        {"cp-exclude": (list, str), "cp-exclude-from-files": (list, str)},
-        filename,
-    )
+    plugin_manager = PluginManager()  # Will edit passed dictionary
+    plugin_manager.config.define_str("job", "ps-format")
```

Comment:
We can remove `define_` prefix at all, but I am not sure:
```
plugin_manager.config.str("job", "ps-format")
plugin_manager.config.str("job", "life-span")
plugin_manager.config.str_list("storage", "cp-exclude")
plugin_manager.config.str_list("storage", "cp-exclude-from-files")
```

Reply:
Let's keep `define_`